.travis.yml

```diff
@@ -40,3 +40,7 @@ matrix:
 
 script:
   - ci/travis.sh
+
+branches:
+  except:
+    - /renovate/*/
```

Comment:
Problem here, actually: if we skip CI for these branches, we'll update to incompatible versions of our dependencies too!

We should just delete the exclusion.  Ideally we would also rewrite our CI scripts to use the pinned versions everywhere and check they're consistent, but you might want to leave that to a future PR (or even push them onto me 😉)

Reply:
The problem is that renovate puts these branches directly into the main repo, so the CI services want to run CI *twice* – once b/c it's a branch in the main repo, and a second time b/c it's a PR. I *think* this should eliminate the first CI, but keep the second?

Reply:
@Zac-HD what @njsmith says. :) Notice that the PR is indeed tested below, but the branch itself is not.

Reply:
...Derp, yeah, that makes sense.  (in my defence- no, wait, pyup does the same 🤦‍♂️)